orienteer-bpm/src/main/java/org/orienteer/bpm/Initializer.java

```diff
@@ -18,6 +18,7 @@ public void init(Application application) {
 	@Override
 	public void destroy(Application application) {
 		OrienteerWebApplication app = (OrienteerWebApplication)application;
+		app.unregisterModule(BPMModule.class);
```

Comment:
I think it is logic, because in method init() we initialize module (register module) and in method destroy we destroy  module (unregister module). It is not required to unregister module here it can be unregistered in other class. If method unregisterModule() is not called module can't be unregister and it works when it must be off.